### PR TITLE
chore: repository governance

### PR DESCRIPTION
**Pull request by segraef**
## Repository governance update

This PR was automatically created by the AVM Team hive-mind using the [grept](https://github.com/Azure/grept) governance tool.

We have detected that some files need updating to meet the AVM governance standards.
Please review and merge with alacrity.

Grept config source: `git::https://github.com/Azure/Azure-Verified-Modules-Grept.git//terraform`

Thanks! The AVM team :heart:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,10 +13,6 @@ The pattern module is designed to be modular and composable. The default deploym
 
 ![AI Reference Implementation Baseline Architecture](./media/ai-reference-implementation-architecture.png)
 
-## Architecture Design Record
```
Comment (martinpeck) [inline README.md:16]: Why was this removed?

**Review by martinpeck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (martinpeck) [reply README.md:16]: Suggest reverting this change, unless it's somehow related to repository governance

**Review by mbilalamjad — APPROVED** [review]
Please ensure all checks pass before merging.

**Comment by segraef** [comment]
Supersceeded by #50

**Pull request closed without merging**